ambition_utils/rrule/forms.py

```diff
@@ -138,6 +136,16 @@ def clean(self):
 
         return cleaned_data
 
+    def clean_byminute(self):
```

Comment:
you can simplify it like
```
return self.cleaned_data.get('byminute', 0)
```
if the field isn't submitted to the form, it will be in cleaned_data as `None`, and if a non-integer value is submitted, it will already raise an error from django's other cleaning that happens before this method gets called. so you should be able to assume either int or None in cleaned_data here.

Reply:
Great suggestion. Thank you.